DOCUMENTATION.md

```diff
@@ -10,83 +29,86 @@ If you want to read more into Flatseal's permissions, you can look at the [Permi
+All system libraries, executables and static data | Toggle | Allow read-write access to the application to system libraries located in `/usr`. Since this directory requires root access to write, the permission will be read-only. | `--filesystem=host-os` and `--nofilesystem=host-os`
+All system configurations | Toggle | Allow read-write access to the application to system configurations located in `/etc`. Since this directory requires root access to write, the permission will be read-only. | `--filesystem=host-etc` and `--nofilesystem=host-etc`
+All user files | Toggle | Allow read-write access to the application to the user directory (`$HOME` or `~/`). | `--filesystem=home` and `--nofilesystem=home`
+Other files | Input | Allow read-write access to the application to the directory you desire. <br /> <br /> For example, you would put `~/games` if you want read-write access to `~/games`. If you want read-only access to `~/games`, then you would put `~/games:ro`. | `--filesystem=[PATH]`, `--filesystem=[PATH]:ro` and `--nofilesystem=[PATH]`
 
 #### Persistent
 
 List of the homedir-relative paths created in the sandbox.
 
 Name | Type | Description | `flatpak-override` equivalent
 --- | --- | --- | ---
-Files | Input | Allow only to the application to have access to the targeted directory while restricting other applications from accessing it. <br /> <br /> Starting from `$HOME`, the targeted directory will be remapped to the application's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla` if the Firefox Flatpak has no write access to `~/.mozilla`. | `--persist=[PATH]`
+Files | Input | Allow only to the application to have access to the targeted directory while restricting other applications from accessing it. <br /> <br /> Starting from the user directory (`$HOME` or `~/`), the targeted directory will be remapped to the application's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla`. <br /> <br /> This is also a technique used to declutter the user directory, as it prevents the application from writing to `~/`. | `--persist=[PATH]`
 
 #### Environment
 
 List of variables exported to the application.
 
 Name | Type | Description | `flatpak override` equivalent
 --- | --- | --- | ---
-Variables | Input | Set an environment variable in the application. | `--env=[VAR]=[VALUE]`
+Variables | Input | Set an environment variable in the application to make it run everytime you launch it. <br /> <br /> For example, adding `MOZ_ENABLE_WAYLAND=1` for Firefox to enable the Wayland back-end. | `--env=[VAR]=[VALUE]`
```

Comment:
Instead of "to make it run everytime you launch it" it should say something like "to make the variable available to application when it runs".